### ISSUE TITLE
CheckboxItem not displaying check icon on Android

### ISSUE THREAD

**Issue by Estevete** [nandorojo/zeego#45]
**Code**
```
<DropdownMenuCheckboxItem
	key="test"
	textValue="TEST"
	value={true}>
	<DropdownMenu.ItemIndicator>
		<CheckmarkIcon />
	</DropdownMenu.ItemIndicator>

	<Text>TEST</Text>
</DropdownMenuCheckboxItem>
```

**Current behavior**
CheckboxItem is not displaying the check icon on Android. On iOS & web it works well

Android:
![image](https://user-images.githubusercontent.com/19273351/234568620-bdce189b-b629-43c0-ab6e-f6f92bc33901.png)

iOS:
![image](https://user-images.githubusercontent.com/19273351/234568849-83cf74ad-c96b-4e2d-8d3f-6b4df61bc468.png)

Web:
![image](https://user-images.githubusercontent.com/19273351/234568928-3aa596c5-99b8-4988-b75f-642f754b5161.png)

**Expected behavior**
CheckboxItem displays the check icon on Android

**Comment by alantoa** [comment]
@Estevete no way because Android doesn't support custom icons yet; it only supports the default Android system icons lol.


![CleanShot 2023-05-12 at 1 33 06](https://github.com/nandorojo/zeego/assets/37520667/0869f601-4708-4ae8-9561-6fac3fc0a50e)

**Comment by nandorojo** [comment]
we should still use a checkmark icon here though right?

**Comment by Estevete** [comment]
> we should still use a checkmark icon here though right?

I think so, yep

**Comment by alantoa** [comment]
yes!

**Comment by Darren120** [comment]
updates?

**Comment by nandorojo** [comment]
Open to a PR here.

**Comment by MarceloPrado** [comment]
I can work on a PR for this - do we have any particular guidance @nandorojo ?

**Comment by nandorojo** [comment]
It would be in `createAndroidMenu`, it’s possible something isn’t passing properly to react-native-menu

**Comment by MarceloPrado** [comment]
@nandorojo done!

**Comment by nandorojo** [comment]
Released in `1.10`: https://github.com/nandorojo/zeego/releases/tag/v1.10.0
